config.xml

```diff
@@ -8,6 +8,7 @@
       Red Hat Mobile
   </author>
   <content src="index.html"/>
+  <access origin="*"/>
   <access origin="https://fonts.googleapis.com"/>
```

Comment:
@vchepeli studio will replace  the `*` in `<access origin="*"/>` with the cloud domain on app creation, that's why the other access tags are needed